common/app/conf/switches.scala

```diff
@@ -228,6 +228,12 @@ object Switches {
     safeState = On, sellByDate = never
   )
 
+  val LiveBlogContributorImagesSwitch = Switch("Feature", "liveblog-contributor-image",
+    "Show contributor byline images in live blog blocks",
+    safeState = On,
```

Comment:
I considered defaulting to an `Off` state but thought that'd disable the feature after every deploy. If that's not the case I'll switch it around. No pun intended.